packages/react/src/css/__tests__/types.test.ts

```diff
@@ -0,0 +1,12 @@
+/** @jsxImportSource @compiled/react */
+import { css } from '@compiled/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
We aren't handling this lint rule very well currently. Looking at the config options we have we can either:

- set an override just for tests to just disable the rule
- have every package declare test deps as dev dependencies
- continue supressing

Any thoughts?

Reply:
#2 sounds like the right thing to me. Is there any disadvantage of doing this apart from receptively specifying this dev dep in every needed package?

Reply:
No disadvantage. I'll pull the pin and update this PR for this new dep.